src/Renci.SshNet/Security/CertificateHostAlgorithm.cs

```diff
@@ -7,12 +10,23 @@ namespace Renci.SshNet.Security
     /// </summary>
     public class CertificateHostAlgorithm : HostAlgorithm
```

Comment:
Does this class have to be public? We do not expose an instance publicly right now.
If not, **RsaCertificate** should also not be public.